### PR TITLE
[ Issue : 8 ] Home Page Design

### PR DESCRIPTION
**Pull request by El-amiir**
A very simple design, for now, we don't have many items yet  to be able to generate a good layout

### CODE REVIEW

**Comment by mohamed-abdul-fattah** [comment]
@El-amiir I updated the base branch from `master` to `loginDesign` because this branch is branched out from the `loginDesign` and the PR holds an already reviewed code. After the update the changed files decreased from 13 to 3 files only.
Make sure to update the base branch before merging from `loginDesign` to `master` if the #11 PR is merged before this one.

File: public/assets/sass/sass-components/home.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.home-month-summary{
+    background-color: #4c4c4c3b;
```
Comment (mohamed-abdul-fattah) [inline public/assets/sass/sass-components/home.scss:2]: Consider adding variables, as this way the app theme would be inconsistent and every page might end up with a different color.

**Review by mohamed-abdul-fattah — APPROVED** [review]
LGTM (Looks Good To Me) 👌🏼. Kindly, check my concern regarding the inline colors.

**Comment by El-amiir** [comment]
> LGTM (Looks Good To Me) 👌🏼. Kindly, check my concern regarding the inline colors.

**Pull request merged**